### PR TITLE
Add :ports config option to jet

### PR DESCRIPTION
**Pull request by pschorf**

### CODE REVIEW

File: src/clj/qbits/jet/server.clj [hunk 1]
```diff
@@ -170,6 +170,7 @@ supplied options:
 
 
 * `:port` - the port to listen on (defaults to 80)
+* `:ports` - additional ports to listen on (defaults to [])
```
Comment (shamsimam) [inline src/clj/qbits/jet/server.clj]: We should have just the one entry `ports`.

**Review by shamsimam — COMMENTED** [review]
Should `ssl-port` also be updated to handle multiple ports?

File: src/clj/qbits/jet/server.clj (hunk 1 above)
Comment (pschorf) [reply src/clj/qbits/jet/server.clj]: Done

**Comment by pschorf** [comment]
Not messing with ssl.

**Review by shamsimam — APPROVED** [review]

**Review by sradack — APPROVED** [review]

**Pull request merged**